content/python/concepts/dates/terms/datetime/datetime.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.date()'
+Description: 'Returns a date in the year-month-day format.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.date()` method returns a date in the year-month-day format.
+
+## Syntax
+
+```pseudo
+datetime.datetime(YYYY, M, D, hh, mm, ss)
+```
+
+The following parameters must be valid when passed into the `.datetime()` method:
+
+| Parameter | Required? | Description                                                                                                   |
+| :-------: | :-------: | ------------------------------------------------------------------------------------------------------------- |
+|  `YYYY`   |  &check;  | The year expressed in four digits.                                                                            |
+|   `MM`    |  &check;  | The month expressed in two digits, ranging from 1 to 12.                                                      |
+|   `DD`    |  &check;  | The day of the month expressed in two digits, ranging from 1 to the number of days in a given month and year. |
```

Comment:
Does the `.date()` function need a table like this?

Reply:
*`.datetime()`

I wanted to try a new way of capturing information about method/function parameters, especially if there are more than, say, five or so. What were your thoughts?

Reply:
I could add one to `dates.md` if that's what you mean?

Reply:
I wanted to try a new way of capturing different bits of syntax info related to parameters for functions/methods.

Reply:
I wanted to try a new way of capturing different bits of syntax info related to parameters for functions/methods.

Reply:
Ye I meant adding it to `date.md`